src/Pages/FridgesList/components/NewFridgeModal.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {useContext, useRef, useState} from "react";
+import {be} from "constants/backendSetup";
+import {AlertContext, UserContext} from "context";
+import UniversalModal from "../../../components/UniversalModal";
+import {TextField} from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import {makeStyles} from "@material-ui/core/styles";
+import {path_list} from "../../../constants/routes";
+import {Redirect} from "react-router-dom";
+
+const addFridge = async (token, data) => {
+    const url = `${be.PLAIN}profile/fridges/`;
+    const headers = {
+        "Authorization": "Token " + token
+    };
+    const res = await fetch(url, {
+        headers,
+        method: "POST",
+        body: new FormData(data)
+    });
+
+    if (res.status === 201) {
+        return await res.json();
+    } else {
+        throw res.status;
```

Comment:
Dodałbym, że jeżeli 401 to przekierowuje na stronę logowania z odpowiednim komunikatem